app/views/jobseekers/email_preferences/edit.html.slim

```diff
@@ -0,0 +1,21 @@
+- content_for :page_title_prefix, t(".page_title")
+
+h1.govuk-heading-l = t(".page_title")
+
+p = t(".description")
+
+= form_with model: @model, scope: :jobseekers_email_preferences_form, url: jobseekers_account_email_preferences_path, method: :patch do |f|
```

Comment:
[Minor] I think changing the variable name to @jobseeker would make this easier to understand